### ISSUE TITLE
run ros node error:OpenCV Error: No CUDA support (The library is compiled without CUDA support) in throw_no_cuda or  3.2.0+dfsg/modules/core/include/opencv2/core/private.cuda.hpp, line 97 terminate called after throwing an instance of 'cv::Exception'

### ISSUE THREAD

**Issue by DandyLorenz** [xuhao1/VINS-Fisheye#19]
@xuhao1 Hello, thank you for your contribution。I running ros node will prompt the following error, making it impossible to continue working，node is ：roslaunch vins fisheye_split.launch config_file:=my_ws/src/VINS-Fusion-Fisheye/config/fisheye_ptgrey_n3/fisheye_cuda.yaml.
this is main error tips:

**Comment by xuhao1** [comment]
Please try the latest version in master branch

**Comment by DandyLorenz** [comment]
> Please try the latest version in master branch

yes,i sure use last version by command: `git clone https://github.com/xuhao1/VINS-Fisheye.git` .so ,Is there any solution,smile

**Comment by xuhao1** [comment]
@DandyLorenz This is actually a hardware driver problem, you may try CPU version instead or fix your hardware.
I will publish a docker for this program later, which may also help.

**Comment by DandyLorenz** [comment]
Ok,thanks, I will continue to pay attention and praise~

**Comment by zhangyangcc** [comment]
@DandyLorenz Hi, i have the same problem. Have you fixed it? Thank you very much!

**Comment by zhangyangcc** [comment]
> @DandyLorenz Hi, i have the same problem. Have you fixed it? Thank you very much!

Hi, for me, this error occurs because the package "cv_bridge" doesn't call the "opencv with cuda support".
I refer to method3 in [this](https://blog.csdn.net/bigdog_1027/article/details/79092263) and modify the "CvbridgeConfig.cmake".
It works!
